app/controllers/unconfirmed_users_controller.rb

```diff
@@ -0,0 +1,8 @@
+class UnconfirmedUsersController < ApplicationController
+  def delete
```

Comment:
1. We need to add the `before { authorize user }` filter here to make sure the policy gets used, right? 
2. The built-in rails action for deleting is `destroy` not `delete` IIRC --- but the HTTP verb is `delete` (not POST) so it gets kind of confusing

Reply:
I'll look into the before_action, as for the HTTP verb, if I change the method to destroy, would that make more sense with POST?  Thanks for the quick response.

Reply:
I think that either
- DELETE verb + #destroy makes sense
- POST verb + #create makes sense... because we're "creating" an instance of the delete, if that makes sense?

Reply:
Reading more closely, HTTP DELETE is supposed to idempotent (because it usually refers to a specific resource, not a collection) so I think if our goal is something that is *not* retryable, then POST makes more sense? 🤷 

I don't feel that strongly which verb it is, I just think we should aim to have the verb + action name match if possible

Reply:
Adding `before_action -> { authorize current_user }` to the controller somehow casues  this error we saw before with UserPolicy:
```
NoMethodError at /remove_unconfirmed_users
undefined method `delete?' for #<UserPolicy:0x00007fcc1a482e50>
```
not sure what's going on...investigating.

Reply:
The NoMethodError is the same as yesterday -- the `UserPolicy` has methods for the built-in Rails actions (`create`, `new`, `destroy`, etc etc) so that's one more reason to use one of those methods as the action name

Reply:
Ok, I think I understand what's going on.  The UserPolicy needed a `delete?` method to tell the app, "this user is authorized to run the delete method on the controller".  I should be able to change the controller method to `destroy` - the User policy already has a `destroy?` method that is true for admins, so this should work.